happy/HappyStateDelete.py

```diff
@@ -193,9 +195,38 @@ def __cleanup_host(self):
         self.__delete_host_bridges()
         self.__delete_host_links()
 
+    def __delete_internet(self):
+        """
+        delete internet isp interface
+        functionality similar to command: happy-internet -d ...
+        """
+        # get network interface
+        iface = subprocess.check_output("route | grep '^default' | grep -o '[^ ]*$'", shell=True)
+        isp = iface.rstrip('\n').rstrip('0123456789')
+
```

Comment:
I don't think this is a great way to implement it.  In keeping with the rest of the architecture, I would suggest: 
1. make sure that all the state required to delete the internet is in the global state accessed via `getGlobal()`
2. iterate over all connections to the internet present in the global state

Reply:
@robszewczyk I have made changes base on your suggestion last week, please help to review again.